### ISSUE TITLE
Accessibility - lack of role=option and aria-selected in year picker

### ISSUE THREAD

**Issue by rafal-warpecha** [Hacker0x01/react-datepicker#3564]
**Describe the bug**
Lack of role=option and aria-selected in year picker

**Expected behavior**
To be consistent within all pickers year picker should have role set to option and have aria-selected attribute.

**Comment by wmaca** [comment]
It seems it was addressed here: https://github.com/Hacker0x01/react-datepicker/commit/4f1de77db3e7109d42fecd6163a55fba53dc16fc
